src/traces/scatter/attributes.js

```diff
@@ -65,6 +65,10 @@ module.exports = {
             'See `y0` for more info.'
         ].join(' ')
     },
+    key: {
```

Comment:
Ha I see. Thanks for the clarification.

Maybe `key` should default to the calcdata indices of the **first** plot call?